unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/scripts/WaveGeneratorPoolHandler.cs

```diff
@@ -105,6 +105,11 @@ public void SetWaveFrequency(float waveFrequency, ulong generatorId)
             _generators[generatorId].WaveFrequency = waveFrequency;
     }
 
+    public float GetWaveLength()

```

Comment:
I would suggest using properties instead of get-methods.
e.g.:
public float WaveFrequency => waveFrequency;